apps/server/src/trpc/list.ts

```diff
@@ -65,35 +65,6 @@ export const list = createRouter()
     },
   })
 
-  /** list users */
-  .query('users', {
-    meta: {
-      openapi: {
-        enabled: true,
-        method: 'GET',
-        path: '/users',
-        tags: ['Users'],
-        summary: 'Get many users',
-      },
-    },
-    input: z.object({
-      id: z.string().uuid(),
-      email: z.string().email(),
-    }),
-    output: z.array(entities.User),
-    async resolve({ input }) {
-      return api.userRepo
-        .find({
-          where: {
-            id: input.id,
-            email: input.email,
-          },
-          relations: api.userRepo.relations,
-        })
-        .then((users) => users.map(flatten.user))
-    },
-  })
-
```

Comment:
1. This endpoint is not used, as it essentially returns 1 user
2. This endpoint requires one of ID or email. From what I understand, it's not possible to correctly implement the input validation here with zod.  [issue](https://togithub.com/colinhacks/zod/issues/61))